src/App.js

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import "./app.css";
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      tasks: []
```

Comment:
great use of an empty array to hold your tasks, good choice as it is easily iterable and doesn't complicate your data.